packages/contracts/deploy/new/40_finalize-management-dao/30_install-multisig-on-management-dao.ts

```diff
@@ -28,11 +32,11 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     }
   }
 
-  const approvers = process.env.MANAGEMENT_DAO_MULTISIG_APPROVERS?.split(
-    ','
-  ) || [deployer.address];
+  const approvers = managementDaoMultisigApproversEnv(network).split(',') || [
+    deployer.address,
+  ];
```

Comment:
Shouldn't we remove `|| [deployer.address];` now?

If we keep it and for production networks, we are setting the approvers to `[deployer.address]` if the env variable is not set.
For local networks, we put something in there. 

I think we put `|| [deployer.address]` here particularly for the local tests (so that they can run on the runner) without putting keys in the Github envs.
Since we now have the `managementDaoMultisigApproversEnv(network)` function, we should be able to remove it if I am not missing something.

Reply:
I think you're correct and it can be removed.